gopro2gpx/gpmf.py

```diff
@@ -28,31 +28,30 @@ def __init__(self, config):
 
         # map some handy shortcuts
         self.verbose = config.verbose
-        self.file = config.file
         self.outputfile = config.outputfile
 
 
-    def readFromMP4(self):
+    def readFromMP4(self, in_file):
```

Comment:
This interface change (and similarly `readFromBinary`) would be a breaking change for any external user of `gpmf.py`.